src/services/payment/payment.service.ts

```diff
@@ -0,0 +1,46 @@
+import 'reflect-metadata'
+
+import { CreatePaymentRequest, Payment } from 'ch-sdk-node/dist/services/payment'
+import { ApiResponse } from 'ch-sdk-node/dist/services/resource'
+import { inject } from 'inversify'
+import { provide } from 'inversify-binding-decorators'
+import { PaymentApiClient } from '../clients/paymentApi.client'
+
+import PaymentMapper from 'app/mappers/payment/payment.mapper'
+import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { PAYMENT_CALLBACK_URI } from 'app/paths'
+import TYPES from 'app/types'
+
+@provide(PaymentService)
+export default class PaymentService {
+
+  public constructor(
+    @inject(PaymentMapper) private mapper: PaymentMapper,
+    @inject(TYPES.CHS_URL) private CHS_URL: string,
+    @inject(TYPES.DISSOLUTIONS_API_URL) private DISSOLUTIONS_API_URL: string,
+    @inject(PaymentApiClient) private client: PaymentApiClient
+  ) {}
+
+  public async getPaymentURL(
```

Comment:
generatePaymentURL is probs better to indicate that some logic had to take place, rather than a simple retrieval